ci/scripts/component.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-permissions-api
+  export MEMONGO_DOWNLOAD_URL=https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.0.23.tgz
```

Comment:
More curiosity than anything - what is this link for?

Reply:
I can answer that. `dp-component-test` uses a library ([memongo](https://github.com/benweissmann/memongo)) to spin up a mongodb instance in memory. This library is no longer maintained and there seems to be an issue with the way it identifies the right mongodb package to download and run. Memongo only supports debian up to version 9 (using mongodb up to 3.6.5) and we run these tests on a debian 10 container. Since memongo allows specifying the version it uses (with this environment variable), someone found out that this ubuntu version would work and hence the need of hardcoding it in these scripts.
We are working on a [new library](https://github.com/ONSdigital/dp-mongodb-in-memory) to replace `memongo`, supporting our versions so this environment variable will not be needed very soon.

However, we have a comment explaining this in all other `component.sh` scripts that might be worth adding here?

```suggestion
# memongo tries to download a binary for debian which doesn't work on the container
  export MEMONGO_DOWNLOAD_URL=https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.0.23.tgz
```

Reply:
Added comment